lib/generators/module/module_generator.rb

```diff
@@ -50,22 +50,24 @@ def create_additional_files
   # :nocov:
   def update_and_install
     # spec helper add group
-    insert_into_file 'spec/spec_helper.rb', "\tadd_group '#{file_name.camelize}',
-      'modules/#{file_name}/'\n", after: "# Modules\n"
+    # Don't add these entries to the files in test env/running specs
 
-    # simplecov add group
-    insert_into_file 'spec/simplecov_helper.rb', "\tadd_group '#{file_name.camelize}',
-      'modules/#{file_name}/'\n", after: "# Modules\n"
+    unless Rails.env.test?
+      insert_into_file 'spec/spec_helper.rb', "\tadd_group '#{file_name.camelize}',
```

Comment:
```suggestion
      insert_into_file 'spec/spec_helper.rb', "  add_group '#{file_name.camelize}'," \
```

All these tabs should be two spaces. Also escape the string as this creates a new line that gets inserted into the spec_helper.